### PR TITLE
Full size network label font

### LINKED ISSUE #28: Increase font size for network usage
Currently the network usage numbers and text for download and upload look tiny, at least on my laptop screen and display settings.

This is a request to use the same font size as the other numbers of CPU and RAM usage percentage shown so it looks uniform and easier to read at a glance.

### PR DESCRIPTION
**Pull request by vjr**
Fix #28

### CODE REVIEW

**Comment by vjr** [comment]
TODO: remove the application css and gresource files too. I am not familiar with this area so @PlugaruT or @andirsun  would you help with this part by adding your commits to this PR?

**Comment by vjr** [comment]
> TODO: remove the application css and gresource files too. I am not familiar with this area so @PlugaruT or @andirsun would you help with this part by adding your commits to this PR?

Hi @PlugaruT and @andirsun  - thanks for review/merge my recent PRs... just a reminder that this one is also there but it looks to me requires some additional work which I need your help because I am not familiar with that area.

**Comment by vjr** [comment]
Nevermind @PlugaruT and @andirsun  - I have done the remaining work in https://github.com/PlugaruT/wingpanel-monitor/pull/30/commits/c18688ee6aa01d1f5b66f8019cbfee7785bd1898

Ready for review and merge!

**Comment by andirsun** [comment]
Thanks @vjr let me test the branch and report back.

**Comment by vjr** [comment]
> Thanks @vjr let me test the branch and report back.

Great, thanks!

File: src/Indicator.vala [hunk 1]
```diff
@@ -43,14 +43,15 @@ namespace WingpanelMonitor {
...
+/*
             var provider = new Gtk.CssProvider ();
             provider.load_from_resource ("/com/github/plugarut/wingpanel-monitor/icons/Application.css");
             Gtk.StyleContext.add_provider_for_screen (
                 Gdk.Screen.get_default (),
                 provider,
                 Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION
             );
-
+*/
```
Comment (andirsun) [inline src/Indicator.vala]: @vjr Could you please remove the unused code ?

**Review by andirsun — CHANGES_REQUESTED** [review]
Code and UX looks good.

File: src/Indicator.vala (hunk 1 above)
Comment (vjr) [reply src/Indicator.vala]: > @vjr Could you please remove the unused code ?

@andirsun  done!

**Review by andirsun — APPROVED** [review]

**Pull request merged**